lib/cocoapods-rome/post_install.rb

```diff
@@ -50,6 +52,17 @@ def xcodebuild(sandbox, target, sdk='macosx')
   Pod::UI.puts "Copying frameworks to `#{destination.relative_path_from Pathname.pwd}`"
```

Comment:
I guess it does say "Built x fws" maybe we should just move the copy one and make it say "Copying x fws"